sqlgen/reflect.go

```diff
@@ -21,8 +21,9 @@ type SelectOptions struct {
 	Where  string
 	Values []interface{}
 
-	OrderBy string
-	Limit   int
+	OrderBy   string
+	Limit     int
+	ForUpdate bool
```

Comment:
very nit: commit message link is to MySQL 8.0 documentation, but I see in previous bug fixes we reference MySQL 5.6 idiosyncrasies. Since MySQL 5.6 will be most relevant to Samsara devs who stumble upon and want to consider using this feature, would you update the link to MySQL 5.6?

Reply:
No worries, that's a good call. 👍